node/src/server-modules/GlideFtOptions.ts

```diff
@@ -118,3 +118,39 @@ export interface FtCreateOptions {
     /** The prefix of the key to be indexed. */
     prefixes?: GlideString[];
 }
+
+/**
+ * Represents the input options to be used in the FT.SEARCH command.
+ * All fields in this class are optional inputs for FT.SEARCH.
+ */
+export interface FTSearchOptions {
```

Comment:
```suggestion
export type FTSearchOptions = {
```

Reply:
What is the correct name: `FTSearchOptions` or `FtSearchOptions`, @prateek-kumar-improving ?

Reply:
It is camel case in Python, FtSearchOptions.

Reply:
FtSearchOptions matches the module GlideFtOptions

Reply:
@Yury-Fridlyand why type?  We're using interfaces everywhere to define these.

Reply:
We mostly use types, but I have no preference.
Please rename.